storybookjs/storybook
# Added parent wildcard sortOrder

Issue: https://github.com/storybookjs/storybook/issues/16573

## What I did
Added ability to have parent `order` wildcard in `storySort`.

For example:
```json
"options": {
  "storySort": {
    "includeNames": true,
    "order": [
      "Introduction",
      "*", ["Introduction", "Examples", "*"]
    ]
  }
}
```

## How to test


If your answer is yes to any of these, please make sure to include it in your PR.

How do you normally add test or live sandboxes for testing?
